### DISCUSSION TITLE
Improving interface of incline quest

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#3364] [Ideas]
What you think about dropping single step from answering incline quest and show two answers directly, instead of popup?

![screenshot-8baff25a](https://user-images.githubusercontent.com/899988/136706541-8b70fa27-d53f-4244-8ee6-4c95f5c8e99b.png)

Reasoning and motivation:

According to my testing right now lane quest interface is really, really confusing.

Once people understand how this is supposed to be used, they have no problem. Most even comment "ops, I was really dumb to not be able to use it". Buuut... Basically everyone struggles with it! Out of 8 people new to app - 6 people were completely stumped, blocked and unable to use this.

Problem was primarily with "what I should click to start answering this question"?
Yes, there are question marks and it blinks a bit but apparently it is not sufficient.

The incline quest is the most problematic - even two people who had no problem with sidewalk quest were confused on the first encounter with `incline` quest.

Again the most problematic part is discovering that you need to click on displayed symbolic way to actually show popup with answers for selecting.

I am opening this discussion instead of making PR as I want to first finish already started work, research MapLibre and this change would be rather time consuming from technical side (for start I need to draw diagram about how current code works, then write version of StreetSideSelectPuzzle displaying two separate rotating ways - or to extend existing one - or to figure out how to fit it twice into answer menu).

**Comment by westnordost** [comment]
I think it would be an improvement. The wording of the description would need to be slightly different I think.

**Comment by matkoniecz** [comment]
Current state of solving #3364

![screen14](https://user-images.githubusercontent.com/899988/137685327-80152f3f-218d-472d-9238-6151cd422986.png)

(right now it works by having two `de.westnordost.streetcomplete.view.StreetSideSelectPuzzle` layouts - and I need to figure out how to make them selectable, in the same way as entries in other quests
![screen15](https://user-images.githubusercontent.com/899988/137685636-d985d16d-e11e-4bc9-987c-49a43ecded79.png)
are selectable. Or replace them with something like `RotatedCircleDrawable` that would be redrawn as rotation changes)

**Comment by matkoniecz** [comment]
Was implemented by Tobias
